TUM Campus App/Room Finder/RoomFinderViewController.swift

```diff
@@ -10,6 +10,7 @@ import UIKit
 import Alamofire
 
 final class RoomFinderViewController: UITableViewController, UISearchResultsUpdating {
+    private let errorMessageLabel = UILabel()
```

Comment:
We have an extension for `UITableViewController` and `UIcollectionViewController` `setBackgroundLabel(with text: String)` to set the background label for error messages.